docs/articles/documentation/guides/advanced-guides/authentication.md

```diff
@@ -1,50 +1,37 @@
 ---
 layout: docs
-title: User Roles
-permalink: /documentation/test-api/authentication/user-roles.html
+title: Authentication
+permalink: /documentation/guides/advanced-guides/authentication.html
 ---
-# User Roles
+# Authentication
+
+TestCafe ships with the [user roles](#user-roles) mechanism that covers most authentication types that require user actions on the page.
+
+You can also test websites that use [HTTP Basic](https://en.wikipedia.org/wiki/Basic_access_authentication) and [NTLM](https://en.wikipedia.org/wiki/NT_LAN_Manager) authentication. See [HTTP/NTLM Authentication](#http-authentication) for more information on how to provide the credentials.
+
+## User Roles
 
 Many test scenarios involve the activity of more than one user. TestCafe addresses these scenarios by providing a convenient way
 to isolate authentication test actions and apply them easily whenever you need to switch the user account.
```

Comment:
For the first sentence, can we say:
Many test scenarios involve activity from more than one user.

Also, please rewrite the second sentence. First, be more specific than "providing" and avoid the use of gerunds (-ing form). Please clarify what you mean by "isolate authentication test actions and apply them easily whenever you need to switch the user account". What exactly is an "authentication test action", for example. Is it a set of actions performed after a user has authenticated under a specific user role? Please clarify.

Reply:
`Authentication test actions` are actions performed to log in (like, entering credentials and clicking 'Sign in'). Maybe, yes, 'isolate and apply actions' sounds strange.

So, what about:

*Many test scenarios involve activity from more than one user. TestCafe allows you to isolate test actions required to authenticate a user (like, enter credentials or click 'Sign in'). During the test, you can switch between user accounts with a single method call.*

Reply:
Sounds good. Let's say:
Many test scenarios involve activity from more than one user. TestCafe allows you to isolate test actions required to authenticate a user (e.g., enter credentials, click 'Sign in'). During the test, you can switch between user accounts with a single method call.